packages/create-react-native-library/templates/native-view-legacy/src/{%- project.name %}ViewNativeComponent.ts

```diff
@@ -0,0 +1,25 @@
+import {
```

Comment:
Let's name this file as `<%- project.name -%>View.tsx` to match the name of the component it exports